ember-app/app/services/filter_groups/milestone.js

```diff
@@ -1,29 +1,50 @@
 import Ember from 'ember';
 
+function attr(modelProp, map) {
+  return Ember.computed("model." + modelProp, {
+    get: function(key){
+      var filters = this.get("model." + modelProp).map(map);
+      filters.insertAt(0, Ember.Object.create({
```

Comment:
I could be missing something, but wouldn't this create a new "No Milestone" filter on every get ?